Cython/Compiler/Nodes.py

```diff
@@ -2520,6 +2531,9 @@ def declare_arguments(self, env):
     def need_gil_acquisition(self, lenv):
         return self.type.with_gil
 
+    def need_gil_drop(self, lenv):
```

Comment:
Also here, "release" seems a more natural name for a lock operation than "drop".